packages/jss-plugin-syntax-rule-value-function/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "jss-plugin-syntax-rule-value-function",
+  "description": "JSS plugin for function syntax",
+  "version": "0.1.0",
```

Comment:
I guess it should correspond the jss version?

Reply:
Lerna will take care of it when we publish them.

Reply:
https://github.com/lerna/lerna#publish